app/server/api/credit_transfer_api.py

```diff
@@ -449,6 +450,28 @@ def post(self):
         }
         return make_response(jsonify(response_object)), 201
 
+class CreditTransferStatsApi(MethodView):
+    @requires_auth(allowed_roles={'ADMIN': 'any'})
+    def get(self):
+
+        start_date = request.args.get('start_date')
+        end_date = request.args.get('end_date')
+        encoded_filters = request.args.get('filters')
+        encoded_filters = encoded_filters.replace(" ", "+")
+        decoded_filters = base64.b64decode(encoded_filters)
```

Comment:
this is probably something we worry about later, but my gut is that we shouldn't need to encode the filters to get them to play nice - query strings are pretty good at handling this sort of thing, and more cannonical IMO. Eg like https://www.carsales.com.au/retailapi/home/v2/carsales/search?makePopular=Mazda&model=3&stateList=South%20Australia&price=%3B60000&&_a=all&_t=price

Reply:
yeah this is definitely unneeded. i had it temporarily to test the parameters serverside where i found the other blocker relating to computed properties but will change the params to separate key-values